sql/schema.sql

```diff
@@ -79,7 +79,7 @@ CREATE TABLE tests_regions (
 CREATE TABLE comparison_types (
   id serial PRIMARY KEY,
   name text NOT NULL,
-  symbol text NOT NULL
```

Comment:
Curious what the situation is for needing symbol nullable but not name. I'm sure it's fine, but just want to know. I was just trying my best guess up front with making it required.